web/src/main/webResources/WEB-INF/web.xml

```diff
@@ -472,8 +472,7 @@
     <session-timeout>${sessionTimeout}</session-timeout>
     <cookie-config>
       <http-only>true</http-only>
-      <!-- Uncomment if using https -->
-      <!--<secure>true</secure>-->
+      <secure>${cookieSecureFlag}</secure>
```

Comment:
I would rename it to `sessionIdCookieSecureFlag`.

Reply:
Why `sessionId`? Maybe `sessionCookieSecureFlag` would be good?